docs/source/secured_private_asset_transfer_tutorial.md

```diff
@@ -0,0 +1,394 @@
+# Using Private Data for secured asset transfer in Fabric
```

Comment:
Private data is only one feature that is used to create the transfer scenario. State based endorsement and access control are also key to how he sample works. I would change the title to something more general.

Reply:
Agreed, updated title.